netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -76,6 +91,28 @@ public partial class CustomAttributeBuilder
         public CustomAttributeBuilder(System.Reflection.ConstructorInfo con, object[] constructorArgs, System.Reflection.PropertyInfo[] namedProperties, object[] propertyValues) { }
         public CustomAttributeBuilder(System.Reflection.ConstructorInfo con, object[] constructorArgs, System.Reflection.PropertyInfo[] namedProperties, object[] propertyValues, System.Reflection.FieldInfo[] namedFields, object[] fieldValues) { }
     }
+    public partial class DynamicILInfo
```

Comment:
I think this type should be sealed

Reply:
It is not sealed in .NET Framework...

Reply:
Yeah. I'd rather not seal it, as it defeats the point of the exercise, which is making more existing code work.